doc/source/how-to-run-simulations.rst

```diff
@@ -1,291 +1,338 @@
-
-1. Have the same Python environment in all nodes.
-2. Have a copy of your code (e.g. your entire repo) in all nodes.
-3. Have a copy of your dataset in all nodes (more about this in :ref:`simulation
-   considerations <considerations-for-simulations>`)
-4. Pass ``ray_init_args={"address"="auto"}`` to `start_simulation
-   <ref-api-flwr.html#flwr.simulation.start_simulation>`_ so the ``VirtualClientEngine``
-   attaches to a running Ray instance.
-5. Start Ray on you head node: on the terminal type ``ray start --head``. This command
+Flower's ``Simulation Engine`` allows you to run FL simulations across multiple compute
+nodes. Before starting your multi-node simulation, ensure that you:
+
+1. Have the same Python environment on all nodes.
+2. Have a copy of your code (e.g., your entire repo) on all nodes.
+3. Have a copy of your dataset on all nodes. If you are using partitions from `Flower
+   Datasets <https://flower.ai/docs/datasets>`_, ensure they are the same.
+4. Start Ray on your head node: on the terminal, type ``ray start --head``. This command
    will print a few lines, one of which indicates how to attach other nodes to the head
    node.
-6. Attach other nodes to the head node: copy the command shown after starting the head
-   and execute it on terminal of a new node: for example ``ray start
-   --address='192.168.1.132:6379'``
+5. Attach other nodes to the head node: copy the command shown after starting the head
+   and execute it on the terminal of a new node. For example: ``ray start
+   --address='192.168.1.132:6379'``.
 
 With all the above done, you can run your code from the head node as you would if the
-simulation was running on a single node.
+simulation were running on a single node. In other words:
+
+.. code-block:: shell
 
-Once your simulation is finished, if you'd like to dismantle your cluster you simply
+    # From your head node, launch the simulation
+    flwr run .
+
+Once your simulation is finished, if you'd like to dismantle your cluster, you simply
 need to run the command ``ray stop`` in each node's terminal (including the head node).
 
-Multi-node simulation good-to-know
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. note::
 
-Here we list a few interesting functionality when running multi-node FL simulations:
+    When attaching a new node to the head, all its resources (i.e., all CPUs, all GPUs)
+    will be visible by the head node. This means that the ``Simulation Engine`` can
+    schedule as many ``ClientApp`` instances as that node can possibly run. In some
+    settings, you might want to exclude certain resources from the simulation. You can
+    do this by appending `--num-cpus=<NUM_CPUS_FROM_NODE>` and/or
+    `--num-gpus=<NUM_GPUS_FROM_NODE>` in any ``ray start`` command (including when
+    starting the head).
 
-User ``ray status`` to check all nodes connected to your head node as well as the total
-resources available to the ``VirtualClientEngine``.
+FAQ for Simulations
+-------------------
```

Comment:
Let's add a FAQ here about running simulations on multiple machines. This is a question I've seen a few times. We can then link it to your explanation above, where we call it multi-node.

Reply:
How about also adding an FAQ about tracking results from simulations?

Reply:
what do you mean by "tracking results" here?